ruy/profiler/instrumentation.cc

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 
 #ifdef RUY_PROFILER
 
+#include <string.h>
```

Comment:
nit: can we include <cstring> instead?  I know that's a purely cosmetic difference.

Reply:
(see other comment, will do myself)